bentoml/lightgbm.py

```diff
@@ -51,7 +56,7 @@ def load(
     tag: str,
     booster_params: t.Optional[t.Dict[str, t.Union[str, int]]] = None,
     model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
-) -> "lgb.basic.Booster":
+) -> LightGBMModelType:
```

Comment:
it should be `t.Union[lgb.basic.booster, _LightGBMModelType]` instead of `t.Union["lgb.basic.booster", _LightGBMModelType]` right? because the `lgb` isn't imported under `if t.TYPE_CHECKING`?

Reply:
you should also have `import lightgbm as lgb` under `t.TYPE_CHECKING` as well